test/e2e/tests/privacy/advanced-functionality-privacy.spec.ts

```diff
@@ -0,0 +1,125 @@
+import assert from 'assert';
+import { Mockttp, MockedEndpoint } from 'mockttp';
+import { withFixtures } from '../../helpers';
+import FixtureBuilder from '../../fixture-builder';
+import HomePage from '../../page-objects/pages/home/homepage';
+import OnboardingCompletePage from '../../page-objects/pages/onboarding/onboarding-complete-page';
+import OnboardingPrivacySettingsPage from '../../page-objects/pages/onboarding/onboarding-privacy-settings-page';
+import {
+  importSRPOnboardingFlow,
+  completeImportSRPOnboardingFlow,
+} from '../../page-objects/flows/onboarding.flow';
+
+async function mockApis(mockServer: Mockttp): Promise<MockedEndpoint[]> {
+  return [
+    await mockServer
+      .forGet('https://token.api.cx.metamask.io/tokens/1')
+      .thenCallback(() => {
+        return {
+          statusCode: 200,
+          json: [{ fakedata: true }],
+        };
+      }),
+    await mockServer
+      .forGet('https://bridge.api.cx.metamask.io/getAllFeatureFlags')
+      .thenCallback(() => {
+        return {
+          statusCode: 200,
+          json: [{ fakedata: true }],
+        };
+      }),
+    await mockServer
+      .forGet('https://on-ramp-content.api.cx.metamask.io/regions/networks')
+      .thenCallback(() => {
+        return {
+          statusCode: 200,
+          json: [{ fakedata: true }],
+        };
+      }),
+    await mockServer
+      .forGet('https://chainid.network/chains.json')
+      .thenCallback(() => {
+        return {
+          statusCode: 200,
+          json: [{ fakedata: true }],
+        };
+      }),
+  ];
```

Comment:
I think we should also add the requests to `https://accounts.api.cx.metamask.io/v2/activeNetworks?`

Currently there's a bug where even if we have the basic functionality off the requests to this endpoint  are sent regardless. It would be great if we capture this here too -- to trigger this request we need to click on the Account

https://github.com/MetaMask/metamask-extension/issues/32312

Reply:
addressed in commit https://github.com/MetaMask/metamask-extension/pull/28110/commits/82dd16a836ec322af9d47fc7211a091a9940ddd2